tests/nlu/extractors/test_crf_entity_extractor.py

```diff
@@ -93,7 +96,7 @@ async def test_train_persist_with_different_configurations(
     pipeline = [
         {"name": "SpacyNLP", "model": "en_core_web_md"},
         {"name": "SpacyTokenizer"},
-        {"name": "CRFEntityExtractor"},
+        {"name": "CRFEntityExtractorGraphComponent"},
```

Comment:
This test failed locally due to `ImportError: Cannot retrieve class from path CRFEntityExtractorGraphComponent.`
I'd appreciate any ideas how to adapt this one 🙏🏼

Reply:
by the way, `test_train_persist_load_with_composite_entities` also needs to be adapted

Reply:
checking why it doesn't work :)

Reply:
This is actually expected. You have migrated the component and it's no longer compatible with the old model architecture. You've to do something similar as Joe did [here](https://github.com/RasaHQ/rasa/blob/5bb331ff7ba75ad564478cbc71fd1a46bce5ee02/tests/nlu/classifiers/test_diet_classifier.py#L122). Or once the `SpacyTokenizer` is also migrated you could just run them one after another

Reply:
Thank you both for your suggestions 🙏🏼  The unit tests have been adapted and are now passing 🎉